### PR TITLE
Sustainability page

### PR DESCRIPTION
**Pull request by Runeii**

### CODE REVIEW

File: src/pages/sustainability/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const Sustainability = ({ intro, content, seoTitle, seoDescription }) => {
+  return (
+    <Layout isNewDesign>
+      <Head description={seoDescription} title={seoTitle} />
+
+      <div
+        className={styles.blurb}
+        dangerouslySetInnerHTML={{ __html: intro }}
+        z
```
Comment (Runeii) [inline src/pages/sustainability/index.js:17]: Stray character

File: src/pages/sustainability/Sustainability.module.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  @mixin body-3;
+  width: 100%;
+  margin-bottom: 80px;
+
+  @media (--tablet) {
+    @mixin body-1;
+  }
+
+  p:not(:last-of-type) {
+    margin-bottom: 1.18em;
```
Comment (Runeii) [inline src/pages/sustainability/Sustainability.module.css:11]: Using such small decimal relative units like this feels confusing. In general the new layout has preferred px over em in most situations – though I can see a case for still using ems if it was a reusable mixin/component that could be used in different font size contexts.

**Comment by Runeii** [comment]
Reviewed, only two small comments on the code.

Visually, on my screen, it looks a little off, with the image/text blocks not alternating betwen left and right.

The space to the left and right is filled with a solid colour. This may be intentional, but also flagging those incase they hadn't been spotted on smaller screens.

image

File: src/pages/sustainability/Sustainability.module.css (hunk 2 above)
Comment (jondujaka) [reply src/pages/sustainability/Sustainability.module.css:11]: The font-size of the `p` tag changes based on breakpoints. Going with an `em` in this case, makes the margin adjust automatically when the `p` fontsize changes.

Defining this in pixels would mean that I would have to adjust the margin also when the breakpoint changes. I can round it up to `1.2` tho, im sure the designers wouldn't mind.

**Comment by jondujaka** [comment]
> Reviewed, only two small comments on the code.
> 
> Visually, on my screen, it looks a little off, with the image/text blocks not alternating betwen left and right.
> 
> The space to the left and right is filled with a solid colour. This may be intentional, but also flagging those incase they hadn't been spotted on smaller screens.
> 
> image

**Pull request merged**